l10n_mx_toponyms/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Mexican Toponyms
+================
+
+This module extends the functionality of Odoo to add Mexican toponyms according with the catalogues publised by SAT on the CFDI 3.3 specification
+
+Currently this catalogues are supported:
+
+* Paises
+* Estados
+* Colonias
+* Localidades (Ciudades)
+* Municipios 
+* CP
+
+
+Installation
+============
+
+Install this module as any other module on your Odoo instance.
+
+
+Configuration
+=============
+
+No special configuration required.
+
```

Comment:
@agb80 Please add the image and link to runbot so anyone can easily try the module:

```

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/193/9.0
```